frontend/src/features/account/LoginErrorPage.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Col, Row } from 'react-bootstrap';
+
+export const LoginErrorPage: React.FC<any> = () => {
```

Comment:
This component name is generic, but the implementation is specific to IE not supported.  Rename, or implement generically.